Mesh_3/include/CGAL/Mesh_3/C3T3_helpers.h

```diff
@@ -3873,7 +3820,11 @@ get_conflict_zone_topo_change(const Vertex_handle& v,
   // Get triangulation_vertex incident cells : removal conflict zone
   // TODO: hasn't it already been computed in "perturb_vertex" (when getting the slivers)?
   // We don't try to lock the incident cells since they've already been locked
+#ifdef CGAL_LINKED_WITH_TBB
+  tr_.incident_cells_threadsafe(v, removal_conflict_cells);
+#else
   tr_.incident_cells(v, removal_conflict_cells);
+#endif
```

Comment:
The test for `CGAL_LINKED_WITH_TBB` is not sufficient. It should be like:
https://github.com/CGAL/cgal/blob/e0330c14006a945ffa75823a37477f69b5792f18/Mesh_3/include/CGAL/Mesh_3/C3T3_helpers.h#L3413-L3424